back/engines/commercial/insights/spec/models/insights/project_spec.rb

```diff
@@ -6,6 +6,6 @@
   subject(:project) { build(:project) }
 
   describe 'associations' do
-    it { is_expected.to have_many(:insights_data_sources).dependent(:destroy) }
+    it { is_expected.to have_many(:insights_data_sources) }
```

Comment:
We could test the deletion of dependent views.

Reply:
Goal is to remove this soon, so won't bother